selfservice/strategy/code/code_recovery.go

```diff
@@ -24,9 +23,9 @@ const (
 )
 
 var (
-	ErrCodeNotFound          = herodot.ErrNotFound.WithReasonf("unknown recovery code")
-	ErrCodeAlreadyUsed       = herodot.ErrBadRequest.WithReasonf("recovery code was already used")
-	ErrCodeSubmittedTooOften = herodot.ErrBadRequest.WithReasonf("The recovery was submitted too often. Please try again.")
+	ErrCodeNotFound          = herodot.ErrNotFound.WithReasonf("unknown code")
+	ErrCodeAlreadyUsed       = herodot.ErrBadRequest.WithReasonf("The code was already used. Please try again.")
```

Comment:
Instead of "Please try again", should we maybe say "Please request another password reset"? Trying the same thing again won't help, right?

Reply:
Yes, good idea.